### DISCUSSION TITLE
Best way to move the Exander Icon (Chevron?) to the left side

### DISCUSSION THREAD

**Discussion by josephnarai** [AvaloniaUI/Avalonia#6712] [Q&A] [answered]
Hi All,

    Hopefully a simple question. I'm porting from an AdonisUI project which has the expander chevron on the left side. The default position in Avalonia is the right. Is there a simple way to move it to the left side?

    Thanks in advance.

**Comment by MarkusKgit** [comment, marked as answer]
You could copy the relevant part from the [default Style](https://github.com/AvaloniaUI/Avalonia/blob/0b9cd49a11c4ef026d2ed6cf07e7aa718ba5a974/src/Avalonia.Themes.Fluent/Controls/Expander.xaml#L90) (Lines 90 to 127) and swap the order of the Items in the Grid (ContentPresenter  would now be Grid.Column="1" and the ExpandCollapseChevronBorder in Grid.Column="0", dito for the ColumnDefinitions)

![Screenshot 2021-10-13 092635](https://user-images.githubusercontent.com/13160892/137086684-891b228c-6d46-4b69-ac00-234ac59b48ef.png)

**Reply by josephnarai** [reply]
Awesome, thank you!
